RecoLocalTracker/SiPixelRecHits/test/BuildFile.xml

```diff
@@ -1,26 +1,20 @@
-<use name="rootmath"/> 
```

Comment:
@fwyzard was it expected? The order is changing

Reply:
I sorted them, since I had to fix this BuildFile by hand